packages/desktopjs/src/Default/default.ts

```diff
@@ -260,6 +260,14 @@ export namespace Default {
             return new DefaultMessageBus(this);
         }
 
+        public setOptions(options: any) {
+            throw new TypeError("SetOptions not implemented by this container");
+        }
+
+        public isAutoStartEnabledAtLogin(): Promise<boolean> {
+            return Promise.reject(new TypeError("Auto start is not implemented by this container"));
```

Comment:
@sophophilic Same as setOptions being a noop, I feel this should return false as it can never be enabled.

Reply:
Agreed and addressed in the latest commit